project1/css/index.css

```diff
@@ -8,115 +8,75 @@ Moving down the .html page, I ceased efforts after attempting to format "Feature
 */
 
 .container {
-	max-width: 950px;
-	margin: 0 auto;
+	width: 850px;
+	margin: 0 auto; /* how to center page w/o using margin? */
 }
```

Comment:
To center without using margins, rather than setting your width and height, you would use percentages of the view screen. We will focus more on that today.